### PR TITLE
Add dev tooling helper script to populate application lifecycle

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Fixes 

https://issues.redhat.com/browse/HAC-2494

## Description

Helper script will create all the custom resources need for the application lifecycle and expects the user to start the build pipeline by merging the PR.

List of resources created:
1. Application 
2. Component
3. Integration Tests (both in application and component context)
4. Static environment
5. Managed environment (ReleasePlan is created)

Note: Since appstudio staging user doesn't have enough permissions to create other resources in managed workspace (different ns) , some parts of the code is commented out. If a user has enough permission, then uncommenting the managed workspace resource creation will create entire release setup.

## Type of change

- [x] Other (please describe): 

Helper script to create a development/testing environment.

## Screen shots / Gifs for design review 

Screenshot 2022-12-07 at 5 25 38 PM

https://user-images.githubusercontent.com/9964343/206175649-bcf91a7f-b09f-4d72-a0d2-e85ad3beba4e.mov

## How to test or reproduce?

**Added Readme:**

## Description

This script helps to setup a development environment for an entire application lifecycle.

### Prerequisites

1. Install the [appstudio-staging-ci](https://github.com/apps/appstudio-staging-ci) Github app in your github 
2. Copy the content from `template.env` file and create a new `.env` file and add your values.

### Setup

1. Run the setup script from your root directory and pass your github repository to build.
   ```sh
   helpers/setup.sh https://github.com/your_username_/Project-Name.git
   ```
2. Once the setup is complete, merge your new PR in your repository to trigger a new build pipeline.

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

cc: @christianvogt

### CODE REVIEW

File: helpers/resources/release/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+create_resources() {
+  oc kustomize "$SCRIPT_DIR/dev-workspace"  | sed 's|<dev-namespace>| '${DEV_WORKSPACE}'|' | sed 's|<managed-namespace>|'${MANAGED_WORKSPACE}'|' | oc apply -f -
+
+  # appstudio-staging users dont have enough permissions for managed namespace, disabling this for now
+  # oc kustomize "$SCRIPT_DIR/managed-workspace" | sed 's|<managed-namespace>|'${MANAGED_WORKSPACE}'|' | sed 's|<dev-namespace>|'${DEV_WORKSPACE}'|'
+}
+
+create_quay_secret() {
+  podman login --username "$QUAY_ROBOT_ACCOUNT" --password "$QUAY_PASSWORD" --authfile "$AUTH_FILE" quay.io
```
Comment (rottencandy) [inline helpers/resources/release/bootstrap.sh:31]: Is `--authfile` option needed here? I think it already uses the same default path for auth when the arg is not given.

**Review by rottencandy — COMMENTED** [review]
Tested the scripts, working fine. Just had a question

File: helpers/resources/release/bootstrap.sh (hunk 1 above)
Comment (karthikjeeyar) [reply helpers/resources/release/bootstrap.sh]: Earlier I was using tmpDir to store the auth creds, but now i am using the default path so this --authfile option is no longer needed. Removed it.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Pull request merged**